javascript/javascript1/week3/class_tasks.js

```diff
@@ -0,0 +1,147 @@
+// const a = [22, 54, 33, 67, 91];
+// a.push(42);
+// a.unshift(43);
+// console.log(a.length);
+// a.shift();
+// a.pop();
+// console.log(a.length);
+// for (let i = 0; i < a.length; i++) console.log(a[i]);
+// for (let item of a) console.log(item);
+// console.log(a.toString());
+// console.log(a.join(" -*-*- "));
+// if (a.includes(33)) {
+//   console.log(
+//     `33 exists at position ${a.lastIndexOf(33) + 1} (human counting from 1)`
+//   );
+// }
+
+classmates = [
+  { name: "vipavee", age: 20 },
+  { name: "Natalia", age: 20 },
+  { name: "Alex", age: 37 },
+  {},
+];
+classmates[1].age++;
+console.log(classmates[1].age);
+
+for (let person of classmates) {
+  if (person.name === "Alex") {
+    console.log(
+      `Alex is found!!! at position ${classmates.indexOf(person) + 1}`
+    );
+    break;
+  }
+}
+
+console.log(Object.keys(classmates[3]));
+
+const cat = {
+  name: "Bertie",
+  breed: "Cymric",
+  color: "White",
+  greeting: function () {
+    console.log("Miow!");
+  },
+};
+const catName = "name";
+console.log(cat[catName]);
+cat.greeting();
+cat.color = "black";
+
+
+const myFavBand = {
+name: "Beatles",
+nationality: "British", 
+genre: "Soft rock"
+members: 4,
```

Comment:
did you miss "," after 'genre'? the color highlight is different. Surely it is going to break the rest of the values after this.

Reply:
Thanks! I couldn't figure out where is the mistake, despite Prettier extension warned me about it ))) I plan to revisit the code with "fresh eyes", but you have been much faster then I did. Thanks once again!